client.go

```diff
@@ -242,6 +247,53 @@ func (c *IClient) SearchWithTag(tag string) (*SearchResponse, error) {
 	return c.parseSearchResponse(resp)
 }
 
+// New Function Search With Title:
+func(c* IClient) SearchWithTitle(title string) (*SearchResponse, error) {
+	request := makeSearchBody(title, "")
+	body, err := json.Marshal(request)
+	if err != nil {
+		log.Printf("IClient.post(%s, %v) returned an error: %v\n", searchEndpoint, body, err)
+	}
+
+	header := make(http.Header)
+	header.Add("accept", "application/json")
+	header.Add("Content-Type", "application/json")
+	if c.Debug {
+		log.Println("----")
+		log.Printf("SearchWithTag: %s %s %v", searchEndpoint, body, header)
+	}
+	// log.Printf("SearchWithTag: %s %s %v", searchEndpoint, body, header)
+	resp, err := c.post(searchEndpoint, bytes.NewReader(body), header)
+
+	if err != nil {
+		if c.Debug {
+			log.Printf("IClient.post(%s, %v) returned an error: %v\n", searchEndpoint, body, err)
+		}
+		return &SearchResponse{}, err
+	}
+
+	return c.parseSearchResponse(resp);
+	
+}
+
+// New Function To upload videos:
+func(c *IClient) UploadAsset(assetID string) {
```

Comment:
Lines 280-296: Remove from this pull request and submit in a separate pull request.

Pull requests are for merging completed code into the main branch.